src/options/OptionHelper.js

```diff
@@ -71,7 +71,9 @@ class OptionHelper {
 
     // Operate on a copy of the plugin, since the webpack task
     // can be called multiple times for one instance of a plugin
-    const instance = Object.create(plugin);
+    const instance = Array.isArray(plugin) ? [] :
+      Object.prototype.isPrototypeOf(plugin) ? Object.create(plugin) :
+      plugin;
```

Comment:
I'm not entirely sure how this is supposed to work but I feel like using an empty array when the option is a non-empty array might break stuff.

I'd actually suggest looking at [the previous implementation](https://github.com/webpack-contrib/grunt-webpack/commit/f0b4e360b6648c363e0b8cb6d9a68d4decf1120e#diff-95b781bf2035f07a2b6b2272cd79f255L65) and only processing `plugins` properties that are directly related to webpack. Recursively processing all `plugins` props is error-prone because one may know when a plugin or loader uses the same property name.